frontend/src/views/supp-allowances/SupplementarySubmitView.vue

```diff
@@ -13,7 +13,7 @@
         </AppButton>
```

Comment:
Most of the comments on panels apply to this component as well.

Reply:
done